### ISSUE TITLE
Methods calling json_encode not returning results

### ISSUE THREAD

**Issue by dfspinto** [orangehrm/orangehrm#489]
Hello,

I've created a VM for testing purposes of orangehrm-4.2.0.1, but all methods that call the php function "json_encode" do not return any result.

Example:

```
public function getSupervisorListAsJson() {

        $jsonArray = array();
        $employeeService = new EmployeeService();
        $employeeService->setEmployeeDao(new EmployeeDao());

        $employeeList = $employeeService->getSupervisorList(true);

        foreach ($employeeList as $employee) {

            $name = $employee->getFirstName() . " " . $employee->getMiddleName();
            $name = trim(trim($name) . " " . $employee->getLastName());
            if ($employee->getTerminationId()) {
                $name = $name. ' ('.__('Past Employee') .')';
            }
            $jsonArray[] = array('name' => $name, 'id' => $employee->getEmpNumber());
        }

        $jsonString = json_encode($jsonArray);

        return $jsonString;
    }
```

![image](https://user-images.githubusercontent.com/3933433/55883627-94262900-5b9e-11e9-87af-2e3a4132b8be.png)

On my local environment, everything works as expected. I've checked database and table schemas, they're the same. 

Hope you have any idea to fix this problem.

Thank you.

**Comment by SharatJanageri** [comment]
Hi @dfspinto , the issue might be related to javascript error, please check .\symfony\web\webres_5fa517042fb5e3.16670395\orangehrmBuzzPlugin\js\buzzCommon.js . For some javascript syntax errors.

**Comment by kavithagurusamy** [comment]
This issue is fixed in OrangeHRM 5.0 version
